app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -726,8 +733,8 @@ public void getLocationFromLocationService() {
     public void onRequestPermissionsResult(int requestCode, String permissions[], int[] grantResults) {
         switch (requestCode) {
             case 1: {
-                if (grantResults.length > 0 && grantResults[0] == PackageManager.PERMISSION_GRANTED ) {
-                    LocationHelper locationHelper = new LocationHelper(MainActivity.this);
+                if (grantResults.length > 0 && grantResults[0] == PackageManager.PERMISSION_GRANTED) {
+                    locationHelper.getLocation();
```

Comment:
I'm sorry. I couldn't understand the need of this call. This is not stored in any variable. Seems wasted?